docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1038,6 +1038,23 @@ How to **investigate**:
   1. The alert fired because of a bug in Mimir: fix it.
   1. The alert fired because of a bug or edge case in the continuous test tool, causing a false positive: fix it.
 
+### RolloutOperatorNotReconciling
+
+This alert fires if the [`rollout-operator`](https://github.com/grafana/rollout-operator) is not successfully reconciling in a namespace.
+
+How it **works**:
+
+- The operator coordinates the rollout of pods between different StatefulSets within a specific namespace and is used to manage multi-zone deployments
+- The operator is deployed in Mimir cells where some services (e.g. ingesters) are deployed in multi-zone
+- The operator reconciles as soon as there's any change in observed Kubernetes resources or every 5m at most
+
+How to **investigate**:
+
+- Check rollout-operator logs to find more details about the error, e.g. with the following Grafana Loki query:
```

Comment:
Is there anything specific we should be looking for?

Reply:
In normal conditions, no error is expected to be logged by the rollout-operator. If anything goes wrong, the logged error should have enough details to better understand what's going on.